.github/workflows/check.yml

```diff
@@ -5,22 +5,26 @@ jobs:
     environment: production
     runs-on: ubuntu-latest
 
+    services:
+      mariadb:
+        image: mariadb:10.5
+        env:
+          MARIADB_ROOT_PASSWORD: root
+          MARIADB_DATABASE: lobsters_dev
+        ports:
+          - 3306:3306
+
     env:
-      DATABASE_URL: mysql2://root:root@localhost/lobsters_dev
+      DATABASE_URL: mysql2://root:root@127.0.0.1/lobsters_dev
```

Comment:
Does this not work for `localhost` still? Interesting!

Reply:
No, then it connects through a socket, which worked before when MySQL wasn't running in a container, but doesn't when it's run in Docker.